app/main/posts/detail/post-video-value.html

```diff
@@ -2,7 +2,7 @@
   <h2 class="form-label">
     <bdi>{{label}}</bdi>
   </h2>
-  <div class="video_embed-fluid">
+  <div class="video_embed-fluid" ng-if="videoUrl">
```

Comment:
so, this was basically causing an issue where you'd see an attempt to load platform, instead of just an empty field